### PR TITLE
Basic Calculator program that can come in CBSE exams

### PR DESCRIPTION
**Pull request by rkraja2751**

### CODE REVIEW

**Comment by rkraja2751** [comment]
Issue #2 : A problem is added from CBSE (old) Syllabus.

File: Basic_Calculator.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+#include <iomanip>
+using namespace std;
```
Comment (sidhyatikku) [inline Basic_Calculator.cpp]: please make it compatible for turbo c++

File: Basic_Calculator.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+if (num2 != 0)
+cout << num1 / num2 << endl;
+else
+cout << "ERROR \nCannot divide by zero" << endl;
+break;
+default:
+cout << "Illegal operation" << endl;
+}
+return 0;
+}
```
Comment (sidhyatikku) [inline Basic_Calculator.cpp]: please add a blank line at the End of file

**Comment by rkraja2751** [comment]
Issue #2 : All the changes expected by you are done.

**Pull request closed without merging**